src/event_loop.rs

```diff
@@ -30,6 +30,9 @@ use crate::{
 /// forbidding it), as such it is neither `Send` nor `Sync`. If you need cross-thread access, the
 /// `Window` created from this `EventLoop` _can_ be sent to an other thread, and the
 /// `EventLoopProxy` allows you to wake up an `EventLoop` from another thread.
+///
+/// ***For cross-platform compatibility, the `EventLoop` must be created on the main thread.*** This
+/// isn't strictly necessary on some platforms, like Windows, but attempting to create
```

Comment:
Looks like this sentence doesn't quite finish?

Great initiative on this PR btw! I have also seen many folks run into this in the past.

Reply:
Thanks for catching that! I've updated the docs to not end in the middle of the